pkg/ebpf/ingress_node_firewall_events.go

```diff
@@ -11,10 +11,14 @@ import (
 	"syscall"
 
 	"github.com/cilium/ebpf/perf"
+	"github.com/google/gopacket"
+	"github.com/google/gopacket/layers"
 )
 
+const maxPacketHeaderSize = 100
```

Comment:
is this also defined in C?

Reply:
yeah the same, I found the binary.read() since it force the user to have fixed size while using 100 worked for ICMP it didn't work for TCP I need to find a better api ideally if it can deal with variable size